compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaParser.java

```diff
@@ -283,6 +283,19 @@ public STNode parseAsFunctionBodyBlock() {
         return funcBodyBlock;
     }
 
+    /**
+     * Completely parses a given input as an object member.
+     *
+     * @return Parsed node
+     */
+    public STNode parseAsObjectMember() {
+        startContext(ParserRuleContext.COMP_UNIT);
```

Comment:
We need to start the relevant contexts ie: we need to start `VAR_DECL_STMT` and `OBJECT_CONSTRUCTOR` contexts